src/webgpu/api/validation/buffer/create.spec.ts

```diff
@@ -46,18 +45,11 @@ g.test('size')
 
 g.test('limit')
   .desc('Test buffer size is validated against maxBufferSize.')
-  .params(u =>
-    u
-      .beginSubcases()
-      .combine('size', [
-        kLimitInfo.maxBufferSize.default - 1,
-        kLimitInfo.maxBufferSize.default,
-        kLimitInfo.maxBufferSize.default + 1,
-      ])
-  )
+  .params(u => u.beginSubcases().combine('sizeAddition', [-1, 0, +1]))
   .fn(t => {
-    const { size } = t.params;
-    const isValid = size <= kLimitInfo.maxBufferSize.default;
+    const { sizeAddition } = t.params;
+    const size = t.device.limits.maxBufferSize + sizeAddition;
```

Comment:
u-nit. regarding https://github.com/gpuweb/cts/pull/3004#discussion_r1336465384, maybe we should use:

```
t.makeLimitVariant('maxBufferSize', { mult: 1, add: sizeAdditon });
```

Just so that if we do end up being able to create a helper util for that kinda of thing, we can easily find this usage and update it?

Reply:
done